tests/unit/test_allocator.cpp

```diff
@@ -428,3 +451,44 @@ TEST_F(AllocatorTest, test_hnsw_reclaim_memory) {
               HNSWFactory::EstimateInitialSize(&params) + hash_table_memory);
     VecSimIndex_Free(hnswIndex);
 }
+
+// TODO: once serializer supports fp64, Add this test to IndexAllocatorTest suite test (or uncomment
+// the serializer related lines in test_hnsw_reclaim_memory)
+TEST_F(AllocatorTest, test_hnsw_reclaim_memory_serializer_info) {
```

Comment:
i took out the lines that test the serializer from test_hnsw_reclaim_memory test, which is templated for both float and doubles, to a regular test for floats because we don't have a serializer for doubles yet

Reply:
but what does it test?

Reply:
@alonre24